src/main.ts

```diff
@@ -114,6 +115,7 @@ async function run() {
             inAppUpdatePriority: inAppUpdatePriorityInt || 0,
             userFraction: userFractionFloat,
             status: status,
+            changesNotSentForReview: JSON.parse(changesNotSentForReview),
```

Comment:
The JSON parsing should be in the validation step - if this is malformed (unparseable) json, we should warn the user